sprint-reports/sprint-2.md

```diff
@@ -0,0 +1,134 @@
+# Sprint N°2 [05/02/2024 - 03/03/2024]
+
+_Start of the project to manage products and categories on the website and application._
+
+## Positive reviews
+
+- Review 1
+
+  - Description: The code has progressed well, and the first presentation of the site and application is very positive.
+  - Commitment:
+    - Said by: Kylian
+    - What: Continuing in this direction.
+    - When: Until the end of the project.
+
+- Review 2
+
+  - Description: Using Figma really helped us to visualise all the pages and to be consistent in the visuals.
+  - Commitment:
+    - Said by: Kylian
+    - What: Continue to use Figma seriously.
+    - When: Until the end of the project.
+
+- Review 3
+
+  - Description: Clear code to help the team progress.
+  - Commitment:
+    - Said by: Barnabé
+    - What: Continue to produce clear, serious code.
+    - When: Until the end of the project.
+
+- Review 4
+
+  - Description: Installation of the ORM is finally complete.
+  - Commitment:
+    - Said by: Cédric
+    - What: Remain determined and motivated.
+    - When: Until the end of the project.
+
+## Negative reviews
+
+- Review 1
+
+  - Description: Poorly organised distribution of tasks.
+  - Consequences: Some tasks are dependent on other tasks, which wastes time in advancing the project.
+  - Commitment:
+    - Said by: Alix
+    - What: Improved management and organisation of task allocation.
+    - When: During the next Sprint (#3).
+
+- Review 2
+
+  - Description: Many pool request are open for too long.
+  - Consequences: Blocking progress on the project.
+  - Commitment:
+    - Said by: Barnabé
+    - What: Be more attentive to pool requests. Sometimes during the week, to check whether there are requests for open pools.
+    - When: During the next Sprint (#3).
+
+- Review 3
+
+  - Description: Incomplete task allocation.
+  - Consequences: Lack of organisation and user stories not assigned at the end of the Sprint.
+  - Commitment:
+    - Said by: Barnabé
+    - What: Allocation of the three user stories to everyone at the start of the Sprint, for better organisation.
+    - When: During the next Sprint (#3).
+
+## Questions
+
+- Question 1
+
+  - Description: Is it possible to increase the maximum number of lines in the linter?
+  - Details: 75 lines instead of 50 lines.
+  - Said by: Alix
+
+- Question 2
+
+  - Description: Is it possible to find another way of changing the rules for the 2 reviewers when the pool request is open for too long?
```

Comment:
```suggestion
  - Description: Is it possible to find another way of changing the rules for the 2 reviewers when the pool request is open for too long? Maybe just adding the possibility to merge despite the fact there are not enough reviewer, because currently we can't
```